static/js/components/email/EmailCompositionDialog_test.js

```diff
@@ -94,6 +100,24 @@ describe('EmailCompositionDialog', () => {
     assert.isTrue(sendStub.called, "called send handler");
   });
 
+  it('should rename "send" button to "Save Changes"', () => {
+    renderDialog(
+      {inputs: {subject: 'abc', body: 'abc'}},
+      {type: AUTOMATIC_EMAIL_ADMIN_TYPE}
+    );
+    assert.equal(getDialog().querySelector('.save-button').textContent, "Save Changes");
+  });
+
+  for (let type of [LEARNER_EMAIL_TYPE, COURSE_EMAIL_TYPE, SEARCH_EMAIL_TYPE]) {
+    it(`should label primary button as"send" for composer type ${type}`, () => {
```

Comment:
Please change the description to `'should show a "Send" label when the dialog is being used to send an email'`